lib/winapi/filemanip.c

```diff
@@ -241,6 +242,130 @@ BOOL MoveFileA (LPCTSTR lpExistingFileName, LPCTSTR lpNewFileName)
     }
 }
 
+BOOL CopyFileA (LPCTSTR lpExistingFileName, LPCTSTR lpNewFileName, BOOL bFailIfExists)
+{
+    static const DWORD readBufferSize = 65535;
```

Comment:
For such constants I prefer constants like 0xFFFF because it will be much more obvious where it originates (not much of an issue for 16 bit, but still; also avoids confusion with 0x10000).

Reply:
Good point, changed.

Reply:
Personally, I'd prefer it like `64*1024`.
As for the buffer size, it currently is only 4KiB (64KiB previously). That's a bit small imho - XDK code seem to copy in 16KiB chunks (at least that's the size Halo 2 uses to copy files), which I think is a reasonable compromise between performance and saving memory.